src/host/cmdline.cpp

```diff
@@ -301,6 +301,95 @@ void SetCurrentCommandLine(COOKED_READ_DATA& cookedReadData, _In_ SHORT Index) /
     cookedReadData.SetBufferCurrentPtr(cookedReadData.BufferStartPtr() + CharsToWrite);
 }
 
+bool IsCommandLinePopupKey(const KEY_EVENT_RECORD& event)

```

Comment:
This used to live in cmdline.cpp, because this file contains the switch-case that handles these exact VKs as well. It should be here so that they're kept in sync.